### ISSUE TITLE
Background transparency doesn't work

### ISSUE THREAD

**Issue by entdark** [JasonStein/Notepads#20]
Probably related to my system version which is Windows 10 1809 17763.557.

**Comment by yaichenbaum** [comment]
1809 supports acrylic, is it turned off in system settings or anything?

**Comment by entdark** [comment]
> 1809 supports acrylic, is it turned off in system settings or anything?

It is turned on and works across the system and other apps but Notepads.

**Comment by JasonStein** [comment]
Another thinig could turn acrylic off is that you are using a laptop with a bttery < 10%

**Comment by entdark** [comment]
> Another thing could turn acrylic off is that you are using a laptop with a bttery < 10%

I am on a desktop PC so it shouldn't be an issue. And as I said it works on other programs.

Maybe you could make some logging system to troubleshoot user problems?
So if the application fails to initialize acrylic on my end it will write that information into a log file and I could send it to you?

**Comment by JasonStein** [comment]
Very interesting, I am not sure why it is failing in your case. The point is that, if acrylic failed, I will not receive any error at all. The feature is completely controlled by the system. That is also why it will turn off when battery is slow. Btw, I am running 17763.593 (yours is  17763.557)

**Comment by entdark** [comment]
I've cloned the source to check out my issues, and transparency started worked out of nowhere.

It's actually my bad, that I told it didn't work, because there were 2 reasons for this:
1. I disabled transparency effect on Windows for some time, but forgot about that.
2. Then I reenabled Transparency effect on Windows but had Tint value in Notepads Settings set to 100%. And there are no transparency when it's set to 100%.

So I set it back to default 70% and it works now.
